core/dbt/config/profile.py

```diff
@@ -44,6 +44,10 @@
 {profiles_file}/profiles.yml
 """.format(profiles_file=PROFILES_DIR)
 
+EMPTY_PROFILE_MESSAGE = """
```

Comment:
Can we just re-use `INVALID_PROFILE_MESSAGE ` here?

We should use the following message in place of the `error_string` in `INVALID_PROFILE_MESSAGE`:

```
The profiles.yml file at {path} is empty
```